nucypher/control/controllers.py

```diff
@@ -302,8 +304,23 @@ def start(self,
     def __call__(self, *args, **kwargs):
         return self.handle_request(*args, **kwargs)
 
-    def handle_request(self, method_name, control_request, *args, **kwargs) -> Response:
+    @staticmethod
+    def json_response_from_worker_pool_exception(exception):
+        json_response = {}
+        json_response['failure_message'] = f"{exception}; " \
+                                           f"{len(exception.failures)} concurrent failures recorded"
+        if exception.failures:
+            failures = []
+            for value, exc_info in exception.failures.items():
+                failure = {'value': value}
+                _, exception, tb = exc_info
+                failure['error'] = str(exception)
```

Comment:
I think this can be done in fewer lines, and without the dict mutation?

Reply:
✔️